.github/workflows/python-test.yaml

```diff
@@ -26,13 +26,21 @@ jobs:
       matrix:
         os: ["ubuntu-20.04", "windows-latest"]
         python_version: ["3.9", "3.10", "3.11", "3.12"]
+    env:
+      UV_SYSTEM_PYTHON: 1
     steps:
         - uses: actions/checkout@v4
         - name: Setup python
           uses: actions/setup-python@v5
           with:
             python-version: ${{ matrix.python_version }}
             architecture: x64
+        - name: Install uv
+          uses: astral-sh/setup-uv@v3
```

Comment:
This is pretty cool - where can I learn more about UV?

Reply:
From my old colleague Charlie Marsh who started it all!
https://talkpython.fm/episodes/show/476/unified-python-packaging-with-uv
Or just the docs:
https://github.com/astral-sh/uv